docs/tools/checkout/configuration.md

```diff
@@ -35,21 +35,14 @@ This URL will redirect members to this page [`https://ouvre-boite.com/`](https:/
 
 The `paywallConfig` is a JSON object which includes a set of customizations for your experience. It includes the following elements:
 
-- `locks` : _required object_, a list of lock objects \(see below\).
+- `locks` : _required object_, a list of lock objects ([see below](#locks)).
 - `title`: _optional string_, a title for your checkout. This will show up on the head.
 - `icon`: _optional string_, the URL for a icon to display in the top left corner of the modal.
-- `callToAction`: _optional object_, a list of messages shown based on the state of the checkout modal \(see below\).
-- `metadataInputs`: _optional array_, a set of input fields [as explained there](./collecting-metadata.md).
-- `persistentCheckout`: _optional boolean_: `true` \_\_if the modal cannot be closed, defaults to `false` when embedded. When closed, the user will be redirected to the `redirect` query param when using a purchase address \(see above\).
-- `useDelegatedProvider`: _optional boolean._ To be announced.
-- `network`: _optional integer._ defaults to `1`. See below.
-- `referrer`: _optional string_. The address which will [receive UDT tokens](../../governance/the-unlock-token/) \(if the transaction is applicable\)
-- `messageToSign`: _optional string_. If supplied, the user is prompted to sign this message using their wallet. If using a checkout URL, a `signature` query param is then appended to the `redirectUri` \(see above\). If using the embedded paywall, the `unlockProtocol.authenticated` includes the `signature` attribute.
-- `pessimistic`: _optional boolean_ defaults to `false`_._ By default, to reduce friction, we do not require users to wait for the transaction to be mined before offering them to be redirected. By setting this to `true`, users will need to wait for the transaction to have been mined in order to proceed to the next step.
```

Comment:
All elements below moved inside the lock!

Reply:
I don't think we handle pessimistic on lock basis.